Lil-Peep/src/styles/modules/site-footer.scss

```diff
@@ -0,0 +1,234 @@
+footer {
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-color: #fff;
+  margin-top: 15px;
+  border-top-style: solid;
+  border-top-color: rgba(28, 26, 26, 0.1);
+  border-top-width: 1px;
+}
+
+.footer-content {
+  margin: 0 40px;
+
+  .social-media-container {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+
+    .social-media-box {
+      display: flex;
+      flex: 1;
+      text-align: center;
+
+      .box-sm {
+
+        .social-media {
+          color: #72808C;
+          font-family: 'Lato', sans-serif;
+          font-size: 18px;
+          line-height: 21px;
+          display: block;
+
+          p {
+            @media screen and (max-width: 768px) {
+              font-size: 14px;
+              text-align: center;
+              width: 100%;
+            }
+          }
+        }
+
+        .social-media-icon-group {
+
+          .sm {
+            display: inline-flex;
+            border-radius: 3px;
+            border: 2px solid #72808c;
+            height: 35px;
+            width: 100px;
+            justify-content: center;
+            align-items: center;
+
+            img {
+              max-height: 20px;
+            }
+
+            @media screen and (max-width: 1024px) {
+              border-radius: 3px;
+              border: 2px solid #72808c;
+              height: 25px;
+              width: 60px;
+            }
+
+            @media screen and (max-width: 768px) {
+              display: inline-flex;
+              height: 35px;
+              width: 85px;
+            }
+          }
+
+          .twt {
+            margin: 0 7px 0 7px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 7px` instead of `0 7px 0 7px`